### PR TITLE
CS Falcon: get detections from right key

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-41622

## Description
in the new API - the  detection ids are under the `alert_ids` in the API response.

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py [hunk 1]
```diff
@@ -5508,12 +5508,13 @@ def get_detection_for_incident_command(incident_id: str) -> CommandResults:
     detection_res = get_detections_by_behaviors(behaviors_id).get('resources', {})
     outputs = []
 
+    # detection_ids are under the alert_ids key in the new (raptor) API, see XSUP-41622
+    detection_ids_key = 'detection_ids' if LEGACY_VERSION else 'alert_ids'
```
Comment (RosenbergYehuda) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:5512]: Are we sure this is related to LEGACY VERSION?
I taught this never worked.

**Review by RosenbergYehuda — COMMENTED** [review]
Nice!
I had one question before approving

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py (hunk 1 above)
Comment (ilappe) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:5512]: according to @meital-aizen -it's look like  yes

**Review by RosenbergYehuda — APPROVED** [review]
Great investigation!

**Review by DeanArbel — APPROVED** [review]
Looks good

File: Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_18.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon
+
+- Fixed an issue where **detection_ids** are missed in the ***cs-falcon-get-detections-for-incident*** command results.
```
Comment (DeanArbel) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_18.md:6]:
```suggestion
- Fixed an issue where **detection_ids** were missed in legacy version ***cs-falcon-get-detections-for-incident*** command results.
```

**Review by TalNos — APPROVED** [review]
Approved for changes in 'Test Playbook - CrowdStrike Falcon - Get Detections by Incident'

**Comment by ilappe** [comment]
Force merged 
[TPB](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/8931707#L702) failed also in [nightly](https://jira-dc.paloaltonetworks.com/browse/CIAC-11870)

**Pull request merged**